src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -216,97 +325,5461 @@ static ColliderJntSphInit D_80B7A888 = {
+                if (D_80B7A68C != 0) {
+                    D_80B7A688 = 1;
+                    D_80B7A68C = 0;
+                }
+                D_80B7E0AC = 0;
+                globalCtx->interfaceCtx.unk_260 = 0;
+            }
+            break;
+
+        case 23:
+            D_80B7A674 = false;
+            if (Actor_HasParent(&this->actor, globalCtx)) {
+                this->unk_15C = 24;
+            } else {
+                func_8002F434(&this->actor, globalCtx, GI_SCALE_GOLD, 2000.0f, 1000.0f);
+            }
+            break;
+
+        case 24:
+            D_80B7A674 = false;
+            if ((func_8010BDBC(&globalCtx->msgCtx) == 6) && (func_80106BC8(globalCtx) != 0)) {
+                if (D_80B7E07C == 0) {
+                    this->unk_15C = 0;
+                } else {
+                    func_8010B680(globalCtx, 0x409C, NULL);
+                    this->unk_15C = 20;
+                }
+            }
+            break;
+    }
+}
+
+static s16 D_80B7AFC4[] = { 0, 1, 2, 2, 1 };
+
+static Vec3f sStreamSoundPos = { 670.0f, 0.0f, -600.0f };
+
+static Vec3s sSinkingLureLocationPos[] = {
+    { -364, -30, -269 },
+    { 1129, 3, -855 },
+    { -480, 0, -1055 },
+    { 553, -48, -508 },
+};
+
+void Fishing_UpdateOwner(Actor* thisx, GlobalContext* globalCtx2) {
+    GlobalContext* globalCtx = globalCtx2;
+    Fishing* this = THIS;
+    Vec3f sp114;
+    Vec3f sp108;
+    Vec3f spFC;
+    s16 headRotTarget;
+    s16 playerShadowAlpha;
+    f32 target;
+    f32 camAtFraction;
+    f32 lureDistXZ;
+    s32 pad;
+    Player* player = PLAYER;
+    Input* input = &globalCtx->state.input[0];
+
+    if (0) {
+        // Strings existing only in rodata
+        osSyncPrintf(VT_FGCOL(GREEN));
+        osSyncPrintf(VT_FGCOL(YELLOW));
+        osSyncPrintf("plays %x\n");
+        osSyncPrintf("ys %x\n");
+        osSyncPrintf(VT_RST);
+    }
+
+    playerShadowAlpha = player->actor.shape.shadowAlpha;
+
+    if ((SQ(player->actor.world.pos.x) + SQ(player->actor.world.pos.z)) < SQ(920.0f)) {
+        Math_ApproachS(&playerShadowAlpha, 0, 1, 40);
+    } else {
+        Math_ApproachS(&playerShadowAlpha, 200, 1, 40);
+    }
+
+    player->actor.shape.shadowAlpha = playerShadowAlpha;
+
+    SkelAnime_Update(&this->skelAnime);
+
+    if ((D_80B7A684 != 0) || (func_8010BDBC(&globalCtx->msgCtx) != 0)) {
+        this->actor.flags &= ~1;
+    } else {
+        this->actor.flags |= 0x21;
+    }
+
+    if ((this->actor.xzDistToPlayer < 120.0f) || (func_8010BDBC(&globalCtx->msgCtx) != 0)) {
+        headRotTarget = this->actor.shape.rot.y - this->actor.yawTowardsPlayer;
+    } else {
+        headRotTarget = 0;
+    }
+
+    if (headRotTarget > 0x2710) {
+        headRotTarget = 0x2710;
+    } else if (headRotTarget < -0x2710) {
+        headRotTarget = -0x2710;
+    }
+
+    Math_ApproachS(&this->unk_164, headRotTarget, 3, 0x1388);
+
+    if (((globalCtx->gameplayFrames & 0x1F) == 0) && (Rand_ZeroOne() < 0.3f)) {
```

Comment:
Im assuming this cant be a `%`

Reply:
Looks like I missed a few that could be `%`. They should all be fixed now